<html>
<head>
</head>
<body>
<h2>ORF predictions</h2>
<p>
To capture the full SARS-CoV-2 coding capacity, we applied a suite of ribosome profiling approaches to Vero cells infected with SARS-CoV-2 for 5 and 24 hours, and Calu3 cells infected for 7 hours. For each time point we prepared three different ribosome-profiling libraries, each one in two biological replicates. Two Ribo-seq libraries facilitate mapping of translation initiation sites, by treating cells with lactimidomycin (LTM) or harringtonine (Harr), two drugs with distinct mechanisms that prevent 80S ribosomes at translation initiation sites from elongating. The third Ribo-seq library was prepared from cells treated with the translation elongation inhibitor cycloheximide (CHX), and gives a snap-shot of actively translating ribosomes across the body of the translated ORF. In parallel, RNA-sequencing was applied to map viral transcripts.
</p>
<p>
The Ribo-Seq data on the SARS-CoV-2 of the 24 hours library does not look as a bona fide ribosome footprint, and therefore was not use in the procedure of the ORFs predictions.
</p>
<p>
The ORFs prediction was done by using two computational tools, PRICE and ORF-RATER, that rely on a different features of ribosome profiling data, and by manual inspection of the data.
</p>
<p>
For more details Finkel, Y., Mizrahi, O., Nachshon, A. et al. The coding capacity of SARS-CoV-2. Nature (2020). https://doi.org/10.1038/s41586-020-2739-1.
</p>
</body>
</html>
